tests/test_env.py

```diff
@@ -100,7 +99,9 @@ def install(self, requirements):
 
 
 @pytest.mark.isolated
-@pytest.mark.skipif(sys.version_info > (3, 6, 5), reason='inapplicable')
-def test_default_pip_is_upgraded_on_python_3_6_5_and_below():
+def test_default_pip_is_never_too_old():
     with build.env.IsolatedEnvBuilder() as env:
-        assert not subprocess.check_output([env.executable, '-m', 'pip', '-V']).startswith(b'pip 9.0')
+        version = subprocess.check_output([env.executable, '-c', 'import pip; print(pip.__version__)']).strip()
+        if hasattr(version, 'decode'):
+            version = version.decode()
+        assert int(version.split('.')[0]) >= 10
```

Comment:
Any reason for not using Version as we do in the code for this check?

Reply:
Not really.